lib/src/test/java/com/auth0/android/lock/LockActivityTest.java

```diff
@@ -106,13 +118,15 @@ public void setUp() {
         when(client.login(anyString(), anyString(), anyString())).thenReturn(authRequest);
         when(client.loginWithOTP(anyString(), anyString())).thenReturn(authRequest);
         when(client.createUser(anyString(), anyString(), anyString())).thenReturn(dbRequest);
-        when(client.createUser(anyString(), anyString(), anyString(), anyString())).thenReturn(dbRequest);
+        when(client.createUser(anyString(), anyString(), any(), anyString())).thenReturn(dbRequest);
```

Comment:
`any()` required for the username parameter, as it could be null or a valid string